src/main/java/com/orbitz/consul/config/ClientConfig.java

```diff
@@ -1,4 +1,20 @@
 package com.orbitz.consul.config;
 
+import com.google.common.base.Preconditions;
+
 public class ClientConfig {
+
+    private CacheConfig cacheConfig;
```

Comment:
`CacheConfig` is only provided using the constructor. This could be final.

Reply:
done.